utils/fake_data.py

```diff
@@ -89,12 +91,31 @@ def safe_street_address(self):
 
     def safe_postal_code(self):
         """
-        Returns a safe postal code as a string.
+        Returns a safe postal code as a string.  Due to swagger fields being treated independently this may not match
+        the random state value in an address object.
         """
         while True:
             state = self.state_abbr(include_territories=False)
             if state != "AK" and state != "HI":
-                return self.postalcode_in_state(state)
+                return random.choice(zipcodes.filter_by(state=state)).get("zip_code")
+
+    def safe_city_state_zip(self):
+        """
+        Returns a combination of a valid city, state, and postal code.
+        :return: tuple
+        """
+        while True:
+            state = self.state_abbr(include_territories=False)
+            if state != "AK" and state != "HI":
+                location = random.choice(zipcodes.filter_by(state=state))
+                return {"city": location["city"], "state": location["state"], "postalCode": location["zip_code"]}
+
+    def safe_country(self):
+        """
+        Returns a safe country as a string. Currently restricting to US based CONUS locations not including territories
+        in the states list.
+        """
+        return "United States of America"
```

Comment:
I was seeing other countries being used with US address fields so I'm restricting this to the US with CONUS only moves.